nzedb/libraries/Forking.php

```diff
@@ -519,6 +520,11 @@ private function fixRelNamesMainMethod()
 				$where = "r.proc_files = 0";
 				break;
 
+			case "uid":
+				$join = "STRAIGHT_JOIN release_unique ru ON ru.releases_id = r.id";
```

Comment:
This can probably be a LEFT JOIN with a WHERE ru.releases_id IS NOT NULL.  STRAIGHT_JOIN is a last resort when the LEFT JOINed table is still being used as the first table in the query.  With it being at best 1-to-1 with releases I'd say this won't fall into the same category like release_files.